ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -773,3 +773,8 @@ literals:
 - zone_air_occupied_heating_temperature_setpoint
 - mixed_air_relative_humidity_sensor
 - mixed_air_dewpoint_temperature_sensor
+
+# Added for TW
+- supply_air_isolation_damper_closed_status
```

Comment:
What are the states for this field and the following field?

Reply:
true or false.